docs/SparkboxPRChecklist.md

```diff
@@ -26,6 +29,37 @@
 - [ ] Is all UI content (labels and other strings, but _NOT_ user generated content) translated?
   - Make use of the `trans` or `t` filter
 
+## Accessibility
+Use this checklist when user-facing (non-admin) code is added.
+
+We will divide the work we do as part of our accessibility checklist into a 30/70 ratio of automated/manual testing, or as close as we can get.
```

Comment:
I am realizing 30/70 time split doesnt make sense here 😂 but I still think that the ratio of time spent on automated vs manual testing should be less to more